docs/docs/config.md

```diff
@@ -138,6 +138,7 @@ client_id                      | STRING                              | unique te
 replica_server_id              | LONG                                | unique numeric identifier for this maxwell instance | 6379 (see notes)
 master_recovery                | BOOLEAN                             | enable experimental master recovery code            | false
 gtid_mode                      | BOOLEAN                             | enable GTID-based replication                       | false
+ignore_producer_error          | BOOLEAN                             | terminate Maxwell on Kafka/Kinesis errors if false  | true
```

Comment:
The `true` case behaviour is self explanatory to some degree. The `false` behaviour needs clarification and attention since it causes service disruption. I'll explain both cases then.  Agree with what you said about `producer_error_action=log|terminate`.